sos/policies/distros/cos.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, sysroot=None, init=None, probe_runtime=True,
         super(CosPolicy, self).__init__(sysroot=sysroot, init=init,
                                         probe_runtime=probe_runtime,
                                         remote_exec=remote_exec)
-        self.valid_subclasses += [CosPolicy]
+        self.valid_subclasses += [CosPlugin]
```

Comment:
curious, the `valid_subclasses` already has `CosPlugin` already at line 36

Reply:
Line 36 needs to be removed, as it gets overwritten by the `super()` in `__init__()` from the base `Policy` class.